src/modules/scraper/entry-point/http/controller.ts

```diff
@@ -228,4 +229,13 @@ export class ScraperController {
   async retryIncompleteDeposits(@Body() body: RetryIncompleteDepositsBody) {
     await this.scraperService.retryIncompleteDeposits(body);
   }
+
+  @Post("scraper/fee-breakdown/backfill")
+  @ApiTags("scraper")
+  @Roles(Role.Admin)
+  @UseGuards(JwtAuthGuard, RolesGuard)
+  @ApiBearerAuth()
+  async backfillFeeBreakdown(@Body() body: BackfillFeeBreakdownBody) {
+    await this.scraperService.backfillFeeBreakdown(body);
```

Comment:
Should we be returning something that has a status?

Reply:
If this method doesn't throw, then NestJS automatically returns a success status code 200/201